airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -322,8 +324,11 @@ public KubePodProcess(KubernetesClient client,
 
     LOGGER.info("Creating pod...");
     this.podDefinition = client.pods().inNamespace(namespace).createOrReplace(pod);
-    waitForInitPodToRun(client, podDefinition);
+
+    // If no files need to be copied, the pod's default lifecycle (init containers are run before normal
+    // containers), is sufficient.
```

Comment:
I'm changing some of the behavior around this in https://github.com/airbytehq/airbyte/pull/3866 because there's a case where the init container reliably completes too quickly to be caught if we aren't waiting for file copies.

Reply:
okay will remove this.